example/pubspec.lock

```diff
@@ -291,4 +284,4 @@ packages:
     version: "2.0.0"
 sdks:
   dart: ">=2.17.1 <3.0.0"
-  flutter: ">=2.0.0"
```

Comment:
What causes this to jump to 3.3?

Reply:
Flutter 3.3 is the latest stable release as well as being required by flutter_map due to having completed the migration to the new image provider API.